Makefile

```diff
@@ -103,8 +103,15 @@ endif
 	.actions/openwhisk-utilities/scancode/scanCode.py --config .actions/ASF-Release.cfg ./
 
 
+### version-check: Check Apache APISIX Dashboard version for release
+.PHONY: version-check
+version-check:
+	./utils/version-check.sh
```

Comment:
Shall we need to pass a variable to run this script?

Reply:
good catch. thanks!

Reply:
fixed. please take a look again when you have time, thanks.